src/components/publishing/image.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from "react"
+import styled from "styled-components"
+import Fonts from "./fonts"
+
+const Figcaption = styled.div`
+  & > p {
+    ${Fonts.unica("s14")}
+    margin: 0;
+    color: #999;
+  }
+`
+
+const CaptionLink = styled.div`
+  ${Fonts.unica("s14")}
+  margin: 0;
+  border-bottom: 1px solid black;
+  cursor: pointer;
+  display: inline-block;
+`
+
+const CaptionContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  margin: 10px 0 10px 0;
+`
+
+const BlockImage = styled.img`
+  display: block;
+`
```

Comment:
What he said 😺 . Without it the inline version gives us a ghost space of something like 3px